js/items.js

```diff
@@ -0,0 +1,33 @@
+let colorsLightArr = ["red", "green", "blue", "white", "grey"];
```

Comment:
this file is what I was describing in the comments on the other PR on your own repo, basically this way you can generate the mappings and all the alias stuff much much easier without writing it by hand.